docker/config.py

```diff
@@ -69,8 +69,10 @@
 # By default, the knowledge repo offers:
 # ['debug', 'oauth2', 'bitbucket', 'github', 'google', 'ldap']
 AUTH_PROVIDERS = ["google"]
-OAUTH_GOOGLE_CLIENT_ID = "<client id>"
-OAUTH_GOOGLE_CLIENT_SECRET = "<client secret>"
+OAUTH_GOOGLE_CLIENT_ID = (
```

Comment:
this could cause security issue, you might need to remove it from your commit.

Reply:
I add this file in gitignore, somehow it does not works looks like.

Reply:
removed